plone/app/contenttypes/browser/folder.py

```diff
@@ -198,6 +216,38 @@ def tabular_fielddata(self, item, fieldname):
             'value': value
         }
 
+    def render_cells(self, item):
+        result = []
+        icon_cell_view = queryMultiAdapter(
+            (item, self.request),
+            name=f"tabular-cell-icon"
+        )
+        if icon_cell_view is not None:
+            icon_cell_view.table_view = self
+            result.append(icon_cell_view())
+        for field in self.tabular_fields:
```

Comment:
For collection `self.tabular_fields` is inherited from `collection.py` and a method, not a property.